include/aws/common/hash_table.h

```diff
@@ -74,16 +75,24 @@ struct aws_hash_element {
     void *value;
 };
 
+enum aws_hash_iter_status {
+    AWS_HASH_ITER_STATUS_DONE,
+    AWS_HASH_ITER_STATUS_DELETE_CALLED,
+    AWS_HASH_ITER_STATUS_READY_FOR_USE,
+};
+
 struct aws_hash_iter {
     const struct aws_hash_table *map;
     struct aws_hash_element element;
     size_t slot;
     size_t limit;
+    /* Holds an enum aws_hash_iter_status */
```

Comment:
Is this comment necessary?

Reply:
Nope. That's from an old version of the code. Removed.